tests/ssl-opt.sh

```diff
@@ -80,12 +80,14 @@ fi
 
 if [ -n "${OPENSSL_NEXT:-}" ]; then
     O_NEXT_SRV="$OPENSSL_NEXT s_server -www -cert data_files/server5.crt -key data_files/server5.key"
+    O_NEXT_SRV_NO_WWW="$OPENSSL_NEXT s_server -cert data_files/server5.crt -key data_files/server5.key"
```

Comment:
Improve the readability. 
```suggestion
    O_NEXT_SRV_EARLY_DATA="$OPENSSL_NEXT s_server -early_data -cert data_files/server5.crt -key data_files/server5.key"
    O_NEXT_SRV_WWW="$OPENSSL_NEXT s_server -www -cert data_files/server5.crt -key data_files/server5.key"
```

Reply:
I am not sure why we define the O_NEXT_SRV_WWW, the O_NEXT_SRV has already covered it.

Reply:
Change O_NEXT_SRV_NO_WWW to O_NEXT_SRV_EARLY_DATA